ManagerLib/UserControlEmulator.cs

```diff
@@ -0,0 +1,225 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Text.Json;
+
+namespace ManagerLib
+{
+    public class UserControlEmulator
+    {
+        private static List<Survey> surveys = new List<Survey> { };
+
+        public void StartUpMenu()
+        {
+            Deserialization();
+            if (surveys.Count == 0 || surveys == null)
+            {
+                Console.ForegroundColor = ConsoleColor.DarkRed;
+                Console.WriteLine("Please contact the survey manager as the repository is empty as it is impossible to get all surveys");
+                Console.ResetColor();
+                Console.ReadKey();
+                Environment.Exit(-1);
+            }
+            MainMenu();
+            while (true)
+            {
+                MenuList();
+                Console.Write("Select -> ");
+                bool input = int.TryParse(Console.ReadLine(), out int MainMenuChoose);
```

Comment:
MainMenuChoose is local variable, it should be named in camelCase (not in PascalCase as you did) according to the naming convention (C# Coding Standards and Naming Conventions)